setup/config/webpack.config.dev.js

```diff
@@ -27,6 +27,7 @@ const config = {
     new webpack.DefinePlugin({
```

Comment:
Not related to this file

Question: This seems to be unrelated to hardware wallet? Shouldnt we merge this into development?